cmd/gidari.go

```diff
@@ -56,24 +56,14 @@ func main() {
 }
 
 func run(configFilepath string, verboseLogging bool, _ []string) {
-	ctx := context.Background()
+	file, err := os.Open(configFilepath)
 
-	bytes, err := os.ReadFile(configFilepath)
 	if err != nil {
-		log.Fatalf("error reading config file  %s: %v", configFilepath, err)
+		log.Fatalf("error opening config file  %s: %v", configFilepath, err)
 	}
 
-	cfg, err := transport.NewConfig(bytes)
+	err = gidari.TransportFile(context.Background(), file)
```

Comment:
@prestonvasquez should we pass verboseLogging to the transportFile? with default value false?

Reply:
Yeah, that seems like a good way to go. Idk how else this can be achieved.